index.bs

```diff
@@ -945,43 +949,80 @@ Receiving datagrams can be achieved by calling
 null values indicating that packets are not being processed quickly enough.
 
 <pre class="example" highlight="js">
-const transport = getTransport();
-const reader = transport.datagramReadable.getReader();
-while (true) {
-  const {value: datagram, done} = await reader.read();
-  if (done) {
-    break;
+async function(url) {
+  const transport = new WebTransport(url);
+  for await (datagram of transport.datagramReadable) {
```

Comment:
I suggest writing `const datagram` to avoid using a global variable. Also below.

Reply:
Good catch, I keep doing that.